src/entity/Source.ts

```diff
@@ -91,6 +104,9 @@ export class Source {
   @Index('IDX_source_handle', { unique: true })
   handle: string;
 
+  @Column({ type: 'jsonb', default: defaultFlags })
+  flags: SourceFlags;
```

Comment:
@sshanzel since this is such a small PR can you just add "real logic" like workers in it as well? Just to make sure we did not miss something here and then require additional migrations in follow up PRs. If you want to chunk it still let's make an epic branch for this series of tasks?

Reply:
I agree making it be merged all at once would allow us to have a better view on this whole series of changes. Though I think part of the reason why the tasks were chunked into smaller ones is to allow working in parallel. From the JIRA ticket, this blocks 3 items, merging this small PR early would allow us to work on those 3 tickets at the same time.

Yeah, maybe we can create an epic branch here in the API.

Reply:
@sshanzel yeah got it, if you can at least hook up graphorm in the PR and at tests for the field and I am fine with merging this as well. You can see example on `flags field` tests in `posts`.

Reply:
Yeah, was thinking that would allow us to unblock FE work too. Sure, let me update those items.